packages-common/pluggable-widgets-generator/generators/app/templates/pluggable/native/emptyTemplateJs/src/utils/common.js

```diff
@@ -0,0 +1,18 @@
+export function flattenStyles(defaultStyle, overrideStyles) {
```

Comment:
Will we ship this file inside tools?

Reply:
I thought we were supposed to deliver it inside mendix lib

Reply:
Hmmm, it could very well be. But i think nothing was yet done for this :) Maybe, but why not inside tools? It is more agile and backwards-compatible. We maintain them and it is easier for us to know what every 3rd party pluggable widget need.

Reply:
Tools or MX lib seems to me both a good place for this helper/utility function, although I like the fact that we can have control over the helper functions, since we know best what widget developers need. @diego-antonelli What do you think?

Reply:
Tools is fine

Reply:
Moved `parseStyle` (web) and native common files content to PIW-tools.